src/js/downloads/platforms.js

```diff
@@ -25,27 +25,113 @@
 
 export const Platforms = {
     spongevanilla: {
+        group: 'org.spongepowered',
         id: 'spongevanilla',
         name: "SpongeVanilla",
         suffix: "Vanilla",
         description: "SpongeVanilla is the implementation of the Sponge API on top of Vanilla Minecraft.",
         recommendation: "Recommended for running plugins without Forge mods.",
-        url: "https://repo.spongepowered.org/maven/org/spongepowered/spongevanilla/"
+        category: createDependencyCategory('minecraft', "Minecraft")
     },
     spongeforge: {
+        group: 'org.spongepowered',
         id: 'spongeforge',
         name: "SpongeForge",
         suffix: "Forge",
         description: "SpongeForge is the implementation of the Sponge API on the Minecraft Forge platform.",
         recommendation: "Recommended for running plugins together with Forge mods.",
-        url: "http://files.minecraftforge.net/maven/org/spongepowered/spongeforge/"
+        category: createDependencyCategory('minecraft', "Minecraft")
     },
     spongeapi: {
+        group: 'org.spongepowered',
         id: 'spongeapi',
         name: "SpongeAPI",
         suffix: "API",
         description: "SpongeAPI is the tool developers use to create plugins for the Sponge platform.",
         recommendation: "Recommended for plugin developers.",
-        url: "https://repo.spongepowered.org/maven/org/spongepowered/spongeapi/"
+        category: {
+            name: 'API',
+            id: 'version',
+            extractVersion(version) {
+                return version.split('.')[0]
+            },
+            forBuildType(build) {
+                return this.extractVersion(build.version)
+            },
+            forProject(project) {
+                return project.versions.map(this.extractVersion)
+            }
+        }
     }
 };
+
+export const BuildTypes = [
+    {
+        name: "Stable",
+        id: 'stable',
+        color: 'primary'
+    },
+    {
+        name: "Experimental",
+        id: 'bleeding',
+        color: 'warning'
+    }
+];
+
+export const Labels = {
+    recommended: {
+        name: "Recommended",
+        color: 'success',
+        title: "Recommended build"
+    },
+    latest: {
+        name: "Latest",
+        color: 'success',
+        title: "Latest build"
+    },
+    unsupported: {
+        name: "Unsupported",
+        color: 'danger'
+    }
+};
+
+export const ArtifactTypes = [
+    {
+        name: "Download",
+        classifier: '',
+        icon: 'fa-download'
+    },
+    {
+        name: "Dev",
```

Comment:
To be honest I should just remove it because this artifact can be only used using a build system. There is no reason why you would want to download it. SF has an additional `dev-shaded` JAR that should have the tooltip though.